Description: bin/autoscan.in: Fix "unescaped left brace" warning from Perl.

Requested by Paul Wise <pabs@debian.org>.  Closes: #818855.
Author: Ben Pfaff <pfaffben@debian.org>
Bug-Debian: https://bugs.debian.org/818855

--- autoconf-2.69.orig/bin/autoscan.in
+++ autoconf-2.69/bin/autoscan.in
@@ -3,7 +3,7 @@
 # @configure_input@
 
 # autoscan - Create configure.scan (a preliminary configure.ac) for a package.
-# Copyright (C) 1994, 1999-2012 Free Software Foundation, Inc.
+# Copyright (C) 1994, 1999-2012, 2016 Free Software Foundation, Inc.
 
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -358,7 +358,7 @@ sub scan_sh_file ($)
     {
       # Strip out comments and variable references.
       s/#.*//;
-      s/\${[^\}]*}//g;
+      s/\$\{[^\}]*}//g;
       s/@[^@]*@//g;
 
       # Tokens in the code.
